app/src/js/app.js

```diff
@@ -2,16 +2,20 @@ internet = require('internet');
 geo = require('geo');
 bluetooth = require('bluetooth');
 
-function openRoutesList(e) {
-  geo.getLocation(function(loc) {
-    internet.getRoutes(loc.coords.latitude, loc.coords.longitude, function(routes) {
-      bluetooth.sendRoutes(routes);
-    })
-  });
-}
-
-Pebble.addEventListener('ready', openRoutesList);
+function onPebbleReady(e) {
+  openRoutesList();
+};
```

Comment:
Is the idea behind this split to enable the reader to get the events in order? That is, "when the Pebble is ready, open the routes list"?